Documentation/developers/debugging.rst

```diff
@@ -116,3 +117,97 @@ Usage example:
   logger()->info("Connected port {0}", port_name);
   logger()->debug("Packet size: {0}", packet.size());
   
+
+
+Debugging using Integrated Development Environment (IDE)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+VS Code 
+***************
+
+When compiling a project, by default the compiler creates all the produced files in the `../build` directory.
```

Comment:
I would rephrase everything this way (but I'm not sure it's correct):

Polycube uses `cmake` to create the compilation environment (Makefiles, etc), which, by default is configured to create the executables in `Release` mode.
To debug your code, you must compile Polycube in `Debug` mode, with the following commands:

============
Additional comment: do we really need to set the breakpoints before compiling?

Reply:
Hi @frisso, yes, it is worth adding these two lines.  I'll update immediately.

Regarding the compilation before setting the breakpoints, I had some doubts about it, indeed now I have done other tests and it works even if they are set during debugging.

 Thank you!